### ISSUE TITLE
Animal Blaster buff

### ISSUE THREAD

**Issue by Sagisolar** [Tslat/Advent-Of-Ascension#458]
So Animal Blasters and Sound Cannons have the same trade pretty much. But there is one big issue with it.
![dafsgghh](https://user-images.githubusercontent.com/44548547/54059908-b836c900-41f2-11e9-8b2b-fdf90fb38fff.png)
![dasfghghj](https://user-images.githubusercontent.com/44548547/54059912-bb31b980-41f2-11e9-891f-789347001020.png)
They may have the same DPS but one consume less energy per DPS and also isn't affected by gravity.
In other words, the Sound Cannon outclasses the Animal Blaster with ease.

I would swap around the consumption rate around with both of them and make Animal Cannon not affected by gravity.

**Comment by Tslat** [comment]
Animal blasters got the big poof in 3.2
